pkg/controller/controller.go

```diff
@@ -497,10 +505,10 @@ func (c *Controller) reconcile(ctx context.Context, req reconcile.Request, obj i
 	}
 
 	if m.GetDeletionTimestamp() != nil {
-		event := "delete"
+		eventName := "delete"
```

Comment:
Unrelated change. My editor complained that the variable name matched an imported package name.